src/main/java/org/gitlab4j/api/models/LdapGroupLink.java

```diff
@@ -0,0 +1,57 @@
+
+package org.gitlab4j.api.models;
+
+import org.gitlab4j.api.utils.JacksonJson;
+
+public class LdapGroupLink {
+
+    // The GitLab REST API documentation (https://docs.gitlab.com/ee/api/groups.html#list-ldap-group-links)
+    // does not specify the attributes which are returned for a LDAP group link :-/.
+    // This are the attributes returned by our GitLab instance (15.10.2-ee).
+    // These seem fine because they match with the attributes which you can pass when adding a LDAP group link :-).
```

Comment:
I appreciate this comment, it is very valuable.

But I think we can remove it from this code base.